src/core_plugins/kibana/public/dashboard/partials/save_dashboard.html

```diff
@@ -18,5 +18,7 @@
       <kbn-info info="Change the time filter to the currently selected time each time this dashboard is loaded"></kbn-info>
     </label>
   </div>
-  <button type="submit" ng-disabled="!opts.dashboard.title" class="btn btn-primary" aria-label="Save dashboard">Save</button>
+  <button type="submit" ng-disabled="!opts.dashboard.title" class="btn btn-primary" aria-label="Save dashboard">
+    {{opts.dashboard.titleChanged() ? 'Save a Copy' : 'Save'}}
```

Comment:
I pushed a change to the dashboard save panel so you can play around with both options.  Ignore the extremely ugly code and style for now - I want to settle on which variation to use before taking time to clean it up.

I understand your reservations about the button, definitely makes sense that a user may be confused about why the button text changed and how to get it back.  I still like it better than the text box - IMO just the lesser of two evils and it's equally unclear that checking the checkbox will enable the text box.  The flow seems extremely random, especially for a brand new user (well, both would seem like pretty random side affects to a new user).  We could add help text when the button changes to make it clearer... maybe a little warning icon and 'Saving a change to the title of your dashboard will create a copy of it. Use the original name if you wish to save the current object without creating a copy. '

Gah, both are pretty darn ugly.  At least whichever we choose will hopefully be a very short lived solution!

I definitely defer to you and the design team for whichever you prefer, so just let me know and I will clean it up and address the rest of the great comments here! :)